plugins/org.genivi.faracon/src/org/genivi/faracon/preferences/PreferencesConstants.java

```diff
@@ -8,17 +8,14 @@
 
     // preference keys
     public static final String P_LICENSE          		= "licenseHeader";
-    public static final String P_OUTPUT_PROXIES   		= "outputDirProxies";
-    public static final String P_OUTPUT_STUBS     		= "outputDirStubs";
-	public static final String P_OUTPUT_COMMON    		= "outputDirCommon";
-	public static final String P_OUTPUT_SKELETON  		= "outputDirSkeleton";
-	public static final String P_OUTPUT_DEFAULT   		= "outputDirDefault";
-	public static final String P_OUTPUT_SUBDIRS   		= "outputSubdirs";
+    public static final String P_OUTPUT_DIRECTORY_PATH	= "outputDirectoryPath";
 	public static final String P_GENERATE_COMMON  		= "generatecommon";
     public static final String P_GENERATE_PROXY    		= "generateproxy";
     public static final String P_GENERATE_STUB     		= "generatestub";
     public static final String P_GENERATE_SKELETON 		= "generateskeleton";
```

Comment:
I guess the P_GENERATE_... options are not needed for faracon and could be removed?